### PR TITLE
SE-2234 Cleaned up SRS example

### PR DESCRIPTION
**Pull request by steco**
Generated a datamap
Added a demo of virtual document
Remove duplicates from data

# Pull Request Checklist

- [ ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: examples/use-cases/valuation/SRS csv Example.ipynb [hunk 1]
```diff
@@ -1,5 +1,36 @@
 {
```
Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    sample_data_result_type = "UnitResult/Custom"
</pre><p>is this a "magic" value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <p>@slemasne what do we need to do to fix this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.        unique_columns = [lm.DataDefinition(address=f"UnitResult/{scope}/{unique_column}", name=unique_column, data_type=data_types[unique_column], key_type="PartOfUnique") 
</pre><p> </p><p>A new user will probably need some explanation of <code>PartOfUnique</code> vs <code>Leaf</code>. Is there still a constraint on the minimum required fields e.g. LUID?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <p>usually worth doing a <code>.head(5)</code> to show what the source data looks like </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        sample_data_map_key = create_data_map(
</pre><p>add some commentary as to what this is doing (or link to a KB article if there is one)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.                           "Currency": "string",
</pre><p>is this the mapping in the file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #22.            srs_id = lm.StructuredResultDataId(
</pre><p>ditto this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/use-cases/valuation/SRS csv Example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #32.                #row_df.columns = [i[i.rfind("/")+1:] for i in df.columns]
</pre><p>delete unused code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by rizwansaeed** [comment]
you will need to close and re-raise the PR against `develop` as per the checklist :) 

also, it is worth adding commentary for any SRS pieces you needed help with

File: examples/use-cases/valuation/SRS csv Example.ipynb (hunk 1 above)
Comment (Seamus-Salt) [reply examples/use-cases/valuation/SRS csv Example.ipynb:1]: <p>Done</p>

Comment (Seamus-Salt) [reply examples/use-cases/valuation/SRS csv Example.ipynb:1]: <p>I have removed this as there is no need to import black into a nb in checked in code</p>

Comment (Seamus-Salt) [reply examples/use-cases/valuation/SRS csv Example.ipynb:1]: <p>KB added</p>

**Comment by Seamus-Salt** [comment]
<p>Yea I am in the process of raising another PR I am just closing the conversations here as a checklist to ensure that I have addressed all comments</p><p> </p><p>I am collating my feedback for SRS at the moment </p>

**Pull request closed without merging**